devsetup/Makefile

```diff
@@ -50,14 +50,16 @@ edpm_compute:
 
 .PHONY: edpm_compute_cleanup
 edpm_compute_cleanup:
-	-sudo virsh destroy edpm-compute-${EDPM_COMPUTE_SUFFIX}
-	-sudo virsh undefine edpm-compute-${EDPM_COMPUTE_SUFFIX}
+	sudo virsh destroy edpm-compute-${EDPM_COMPUTE_SUFFIX}
+	sudo virsh undefine edpm-compute-${EDPM_COMPUTE_SUFFIX}
```

Comment:
The dash ("-") in front of those commands meant they could fail (non-zero exit) and make wouldn't complain. I think we still need that, or at least a check to see if the vm's are running before we destroy and undefine them.

Reply:
Thank you for highlighting the use of dash here. I have added it back in the new commit.